src/minecraftServers/Repositories/IServersRepository.cs

```diff
@@ -1,20 +1,22 @@
 ﻿namespace MinecraftServers.Repositories;
 
+using MinecraftServers.Dto;
 using MinecraftServers.Models;
 
 public interface IServersRepository
 {
-    List<Server> GetAll();
+    IEnumerable<Server> GetAll();
 
-    bool Delete(uint id);
+    bool Delete(int id);
 
     Server Add(Server server);
 
     Server GetByIp(string ip);
 
     Server GetName(string name);
 
-    Server GetById(uint id);
+    Server GetById(int id);
```

Comment:
Methods signature in abstraction is different from implementation signature. All method in class that implements that interface return `Server?`